contrib/awslambda/python/src/python/pants/contrib/awslambda/python/subsystems/lambdex.py

```diff
@@ -0,0 +1,13 @@
+# coding=utf-8
+# Copyright 2018 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from pants.backend.python.subsystems.python_tool_base import PythonToolBase
+
+
+class Lambdex(PythonToolBase):
```

Comment:
Would love to see convergence of `PythonToolBase` with `pants.binaries.executable_pex_tool.ExecutablePexTool`, especially since the latter happens to already handle creating the pex itself. I can make a separate PR merging `ExecutablePexTool` into `PythonToolBase`, not sure if there's a utility to a separate `PythonToolBase` that I'm missing.

Reply:
Blargh. Forgot about that. Yea.

Reply:
Yeah, I missed that. That's going to have to be a separate change. I'll tackle it.

Reply:
Issue here: https://github.com/pantsbuild/pants/issues/6888